components/library/jxrlib/history

```diff
@@ -0,0 +1 @@
+library/jxlib@0.2.4,5.11-2022.0.0.0 library/jxrlib
```

Comment:
When you obsolete a package make sure that its COMPONENT_REVISION here is higher than the last released one:
library/jxlib@0.2.4,5.11-2022.0.0.1 library/jxrlib